src/otx/core/model/module/instance_segmentation.py

```diff
@@ -161,3 +161,11 @@ def test_step(self, inputs: InstanceSegBatchDataEntity, batch_idx: int) -> None:
     def lr_scheduler_monitor_key(self) -> str:
         """Metric name that the learning rate scheduler monitor."""
         return "train/loss"
+
+    def _get_state_dict_from_ckpt(self, ckpt: dict) -> dict:
```

Comment:
```suggestion
    def _load_from_prev_otx_ckpt(self, ckpt: dict) -> dict:
```

Please add an unit test for each task.

Reply:
Done, https://github.com/openvinotoolkit/training_extensions/pull/2812/commits/807b0dc82b3807baf40979aa6f704c81740e9591.